projects/fhi-angular-highcharts/CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-# 0.7.0
+# Unreleased
+
+> Sep 08, 2023
+
+* :bug: **Bugfix** Allow dependency `date-fns` to be installed with latest version
```

Comment:
Should this be part of the dependency matrix?

Reply:
I don't think that's necessary, it's one of the dependencies I regard as part of the sentence below the matrix